staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -808,6 +808,31 @@ func (c *Cacher) dispatchEvents() {
 	}
 }
 
+func (c *Cacher) setCachingObjects(event *watchCacheEvent) {
```

Comment:
can you make this a function rather than a member method to make it clear `c` is not being modified?